types/bootstrap/js/dist/carousel.d.ts

```diff
@@ -98,7 +98,7 @@ declare namespace Carousel {
          *
          * @default false
          */
-        slide: string | boolean;
+        ride: string | boolean;
```

Comment:
```suggestion
        ride: 'carousel' | boolean;
```
can we update this as well? (there is no other options support, it reflects data attrib configuration)

Reply:
I also changed the values for `pause`. Not 100% sure, but looks like this if you read the docs